### PR TITLE
Update "advanced mode" info box with network info

### PR DESCRIPTION
**Pull request by roman-drozd-it**

### CODE REVIEW

File: app/src/components/VideoContainers/VideoView.js [hunk 1]
```diff
@@ -244,25 +278,74 @@ class VideoView extends React.PureComponent
...
+										<span className={'RecvBps'}>
+											{(netInfo.recv.sendBitrate/8/1024/1024).toFixed(2)}MB/s
+										</span> 
+										<span className={'RecvSum'}>
+											{(netInfo.recv.bytesSent/1024/1024).toFixed(2)}MB
+										</span>
+
+										<span className={'SendL'}>Send: </span>
+										<span className={'SendBps'}>
+											{(netInfo.send.recvBitrate/8/1024/1024).toFixed(2)}MB/s
```
Comment (misi) [inline app/src/components/VideoContainers/VideoView.js:321]: Use bit not Byte  so please remove /8

File: app/src/components/VideoContainers/VideoView.js [hunk 2]
```diff
@@ -244,25 +278,74 @@ class VideoView extends React.PureComponent
...
 							}
 
-							{ (videoVisible && videoWidth !== null) &&
-								<p>{videoWidth}x{videoHeight}</p>
+							{ isMe && 
+									(netInfo.recv && netInfo.send && netInfo.send.iceSelectedTuple) &&
+									<React.Fragment>
+										<span className={'RecvL'}>Recv: </span>
+										<span className={'RecvBps'}>
+											{(netInfo.recv.sendBitrate/8/1024/1024).toFixed(2)}MB/s
```
Comment (misi) [inline app/src/components/VideoContainers/VideoView.js:313]: Use bit not Byte so please remove /8

Comment (roman-drozd-it) [reply app/src/components/VideoContainers/VideoView.js]: ok, done.

**Comment by roman-drozd-it** [comment]
done.

**Comment by havfo** [comment]
Nice!

**Pull request merged**